dashboard/test/models/script_test.rb

```diff
@@ -1436,6 +1436,21 @@ def populate_cache_and_disconnect_db
     assert_equal [script], scripts
   end
 
+  test "self.valid_scripts: omits pilot scripts" do
+    teacher = create :teacher
+    levelbuilder = create :levelbuilder
+
+    pilot_script = create :script, name: 'pilot-script', pilot_experiment: 'my-experiment'
+    assert_equal true, pilot_script.hidden
+    assert_equal true, has_pilot_script?(Script.all)
```

Comment:
Nit throughout: Using `assert_equal (true|false), <expression>` instead of `(assert|refute) <expression>`.

That, combined with dropping the `has_pilot_script?` helper, makes a few of these assertions extremely compact and expressive:
```rb
assert Script.any?(&:pilot?)
refute teacher_scripts.any?(&:pilot?)
```

Reply:
I'm pretty sure that's an antipattern 😆nice catch.